scripts/toolbox/README.md

```diff
@@ -0,0 +1,39 @@
+# Python Developer Toolbox
+
+Python based Toolbox for various development tasks
+
+## Description
+
+This project started to bundle some development helpers into a single toolbox.
+
+## Getting Started
+
+### Dependencies
+
+* Python3
+* Go
+
+### Installing
+
+* Currently, no specific installation is needed. Just clone the repository and run the scripts.
+
+### Executing program
+
+* Example of how to run the program
+
+```
+python3 main.py --mode=single --init=true --startupParameters='{"rocksdb.block-cache-size": "133713371337"}'
```

Comment:
While trying this command on my machine, I get in _arangodb.log_:
`FATAL [d54b7] {general} cannot convert username 'arangodb' to numeric uid`

The output is:
```
Started ArangoDB in single mode, logfile: work/arangodb.log, port: 8529 (silently!)
--> Started test: "RocksDBCacheSizeTest". Output file is: "work/feed.json"
exit status 4
=> All tests finished
Stopping ArangoDB...
```
There is no feed.json output. Perhaps I am missing some local configuration. I don't mind it, but this is just to let you know that for me it doesn't work out-of-the-box.

Reply:
Resolved :)